test/evergreen.yml

```diff
@@ -1783,6 +1808,32 @@ tasks:
             set -o verbose
 
             ./time_shift_test.sh /usr/local/lib/faketime/libfaketimeMT.so.1 0-1 2>&1
+  
+  - name: format-stress-sanitizer-test
+    #set a 25 hours (25*60*60 = 90000 seconds) timeout
+    exec_timeout_secs: 90000
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger address sanitizer"
+      - func: "format test script"
+        vars:
+          test_env_vars: ASAN_OPTIONS="detect_leaks=1:abort_on_error=1:disable_coredump=0" ASAN_SYMBOLIZER_PATH=/opt/mongodbtoolchain/v3/bin/llvm-symbolizer
+          # run for 24 hours ( 24 * 60 = 1440 minutes), don't stop at failed tests, use default config
+          format_test_script_args: -t 1440
```

Comment:
Is `-n 10` missed in the args setting?

Reply:
Not sure why `-n 10` would be needed here?

Reply:
The Jenkins job runs the test in a for loop for 10 times, and I believe the `-n 10` will help to achieve that. Without `-n` setting it will run as many times as possible (i.e. no limit).

Reply:
We are using -t option to limit the runtime of this task to 24 hours. The `format.sh` script will keep running for 1440 mins (24 hours), keep spawning new job when old ones complete, and then exit gracefully. The `-n` options dictates `format.sh` to run for 10 times and exit which we don't want. The loop in original Jenkins job runs for 10 times and then the job itself get re-triggered. We are using `-t` options of `format.sh` to simulate the for loop in original Jenkins job and the self trigger mechanism.

Reply:
> The -n options dictates format.sh to run for 10 times and exit which we don't want.
I think `-n` only controls the number of run times in each round. How many rounds of testing to be executed is controlled by the `-t` setting (or the default timer value is `-t` is not specified). 

Anyway, from realizing continuous testing perspective it's not that critical to have `-n` setting. The only benefit of setting `-n 10` is to make each round of testing to execute the same number of runs as what Jenkins does today.  I'm okay with the current implementation.

Reply:
I believe if both `-t` and `-n` options are given to `format.sh` as input arguments, the script either spawns `n` number of jobs and exit when these job finish, or, runs until `t` minutes have elapsed and spawning as many job as required given whichever condition get met first.